CMakeLists.txt

```diff
@@ -300,12 +373,13 @@ macro(MACRO_CORE SCRIPT DEFINE BUILD_DEPRECATED)
         ${TIC80CORE_DIR}/core/draw.c
         ${TIC80CORE_DIR}/core/io.c
         ${TIC80CORE_DIR}/core/sound.c
-        ${TIC80CORE_DIR}/api/js.c 
+        ${TIC80CORE_DIR}/api/js.c
         ${TIC80CORE_DIR}/api/lua.c 
         ${TIC80CORE_DIR}/api/moonscript.c 
         ${TIC80CORE_DIR}/api/fennel.c 
         ${TIC80CORE_DIR}/api/wren.c 
         ${TIC80CORE_DIR}/api/squirrel.c
+        ${TIC80CORE_DIR}/api/mrb.c
```

Comment:
Could we name this `mruby` since that seems to be the name used everywhere else?